samples/_run_sample.py

```diff
@@ -0,0 +1,61 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""For development purposes only. Serves as a way to automate running of samples."""
+
+import os
+import sys
+import re
+import subprocess
+import tempfile
+import argparse
+
+STORAGE_ACCOUNT_NAME = os.environ.get("AZSTORAGETORCH_STORAGE_ACCOUNT_NAME")
+CONTAINER_NAME = os.environ.get("AZSTORAGETORCH_CONTAINER_NAME")
+
+
+def modify_url(path):
+    if not STORAGE_ACCOUNT_NAME or not CONTAINER_NAME:
+        print(
```

Comment:
Instead of printing out the error message, let's raise a `ValueError`. This is helpful because it will make sure we short circuit the rest of the script and return a non-zero error code.